make-release.sh

```diff
@@ -126,6 +171,9 @@ git commit -asm "${COMMIT_MSG}"
 # now update ${BASEBRANCH} to the new snapshot version
 git checkout "${BASEBRANCH}"
 
+# update template in the branch
+update_issue_template "${VERSION}" "${ISSUE_TEMPLATE_FILE}"
```

Comment:
is the issue template file in the branch ever actually used? Can you open an issue using the branch's template?

Reply:
When cutting a .z release it won't update the template BTW (no-op)

but then for .0 release it will trigger the PR to merge to main branch

Reply:
script is creating PR to update main from a branch so we need to update the branch if we want to have the template being modified in main branch through this PR